app/src/main/res/drawable/pause_with_background.xml

```diff
@@ -8,8 +8,10 @@
             <size
                 android:width="24dp"
                 android:height="24dp" />
-            <solid android:color="?above" />
-            <stroke android:color="?above"
+            <!-- Vertical pause lines -->
+            <solid android:color="@color/tab_tray_item_media_stroke" />
```

Comment:
Are these the same colors in every theme?

Reply:
Looks good. Can you update the colors to the following:
Light theme:   <color name="ink_50">#312A65</color>
Dark + private theme:  <color name="violet_50">#9059FF</color>

Thanks!